### PR TITLE
Append feedback request comment when there are negative comments

### PR DESCRIPTION
**Pull request by angelikatyborska**
The platform doesn't direct students to give feedback about the automated analysis in any way. Most students won't know how to report an analyzer bug in a way that will lead to a resolution. I am assuming most of them ignore weird analysis results and some of them ask for mentoring (and then the mentor might or might not report the bug to us).

To increase the quality of the analysis, I would rather redirect students to create issues in this repository.

### CODE REVIEW

File: lib/elixir_analyzer/submission.ex [hunk 1]
```diff
@@ -93,6 +93,26 @@ defmodule ElixirAnalyzer.Submission do
...
+    frequencies = get_type_frequencies_in_comments(comments)
+
+    comments =
+      if frequencies.essential + frequencies.actionable > 0 do
+        comments ++
+          [
+            %{
+              type: :informative,
+              comment:
+                "Does this automated feedback look wrong to you? Please [open an issue in the `exercism/elixir-analyzer` repository](https://github.com/exercism/elixir-analyzer/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc)."
```
Comment (angelikatyborska) [inline lib/elixir_analyzer/submission.ex:106]: This can probably be phrased in a nicer way...

Comment (jiegillet) [inline lib/elixir_analyzer/submission.ex:106]: Maybe this?
```suggestion
                "If this automated feedback doesn't look right, please [open an issue in the `exercism/elixir-analyzer` repository](https://github.com/exercism/elixir-analyzer/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc)."
```

**Review by jiegillet — APPROVED** [review]
Looks great!
Hopefully we can get more feedback :)

**Comment by angelikatyborska** [comment]
I'm going to guess it was a mistake to skip the website-copy repo 🙂

**Pull request merged**